example-config.yml

```diff
@@ -130,6 +130,38 @@ itinerary:
   # (Requires using LineItinerary.)
   showRouteFares: false
 
+# Provide a list of transit operators to enhance the user experience for multi-
+# agency implementations. The `order` value of each agencies will dictate the
+# order that routes appear in the route and stop viewers. Also, optionally
+# provide operator logos to appear in itineraries). Note: these logos will
+# override the values found in agency.txt#agency_branding_url. Agencies are
+# matched according to both the feedId and agencyId. This works best with the
+# IBI fork of OTP which returns the necessary information in the route index
+# API. Check the build logs of OTP to see what feedIds and agencyIds were found
+# as these IDs can sometimes be auto-generated.
```

Comment:
What about feeds that have a single agency and no `agency_id`?

Reply:
Additional comments added in 98ef742fc280ac2ea58a2faae8fc09220dbe7c11